src/interfaces/IEntryPoint.sol

```diff
@@ -19,15 +16,6 @@ interface IEntryPoint is INonceManager {
      */
     event UserIntentEvent(bytes32 indexed intentHash, address indexed sender, address indexed submitter);
 
-    /**
-     * An event emitted if the UserIntent part of the solution reverted
-     * @param solIndex - index into the array of solutions to the failed one.
-     * @param intIndex - index into the array of intents to the failed one.
-     * @param segIndex - index into the array of intent segments to the failed one.
-     * @param revertReason - the return bytes from the (reverted) call.
-     */
-    event UserIntentRevertReason(uint256 solIndex, uint256 intIndex, uint256 segIndex, string revertReason);
```

Comment:
Revert reason logging was removed. Since we now expect solutions to be proposed by sophisticated MEV searchers who will be dry running their transactions before submission there is no need to worry about one solution in a list of solutions from failing. It should now be assumed that the whole transaction does not have a solution that could fail